### PR TITLE
Support passing values to HelmReleases

### LINKED ISSUE #3: I can customize the values of my profile with a helm values file
When deploying a Helm chart from a profile, I want to be able to provide configuration for the deployed chart.

```yaml
apiVersion: ...
kind: ProfileSubscription
spec:
   profileURL: https://github.com/testing/testing.git
   values: # This should be taken from the config for the Nginx chart and applied to the HelmRelease
      foo: value1
      bar:
      baz: value2
      oof:
      - item1
      - item2
   valuesFrom:
   - configMapKeyRef:
       # Name of the config map
       name: default-values  # mandatory
       # Namespace of the config map
       namespace: my-ns      # optional; defaults to HelmRelease namespace
       # Key in the config map to get the values from
       key: values.yaml      # optional; defaults to values.yaml
       # If set to true successful retrieval of the values file is no
       # longer mandatory
       optional: false       # optional; defaults to false

  
```

This should work in Fluxv2.

Documentation for configuring HelmReleases https://github.com/fluxcd/helm-controller/blob/main/docs/spec/v2beta1/helmreleases.md#values-overrides

This is not concerned with Secrets or ConfigMaps, if a value is exposed by the Helm chart, this can configure it.

### PR DESCRIPTION
**Pull request by cPu1**
### Description

Allows passing values to a `HelmRelease` by specifying the values in a `ProfileSubscriptionSpec` via the `Values` and `ValuesFrom` fields. 

Closes #3 

TODO:
- [x] Add tests

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

**Comment by Callisto13** [comment]
Could you update or add an example in `examples/` to show how to use the `values` and `valuesFrom` in a profile sub?

**Review by Callisto13 — CHANGES_REQUESTED** [review]
👍  looks good!
Just needs some tests to show values set on subscription end up somewhere.

**Comment by cPu1** [comment]
> Could you update or add an example in `examples/` to show how to use the `values` and `valuesFrom` in a profile sub?

Added an example for both `values` and `valuesFrom` with a `ConfigMap`.

File: pkg/profile/artifact.go [hunk 1]
```diff
@@ -68,21 +61,23 @@ func (p *Profile) createHelmRelease(ctx context.Context) error {
...
 					SourceRef: helmv2.CrossNamespaceObjectReference{
-						Kind:      gitRepositoryKind,
+						Kind:      sourcev1.GitRepositoryKind,
 						Name:      p.makeGitRepoName(),
 						Namespace: namespace,
 					},
 				},
 			},
+			Values:     p.subscription.Spec.Values,
+			ValuesFrom: p.subscription.Spec.ValuesFrom,
```
Comment (aclevername) [inline pkg/profile/artifact.go:80]: can you update the `artifact_test.go` to test these fields are set?

Comment (cPu1) [reply pkg/profile/artifact.go:105]: Done

**Review by Callisto13 — APPROVED** [review]

**Pull request merged**